NEWS.rst

```diff
@@ -1,6 +1,16 @@
 History
 =======
 
+0.11 (2021-05-06)
+-----------------
+* Add asynchronous RDBWriter class (#108)
+* Use a transaction for get_range instead of Lua: faster on server side (#110)
+* Multiplex aio pubsubs over pool's standard connection (#113)
+* Require hiredis for speed and aioredis<2 for compatibility (#114, #118)
+* Improve unit tests, especially async memory backend (#107, #111, #112, #116)
```

Comment:
#107 isn't really about unit testing, it's about supporting redis-py 3.5 without DeprecationWarnings.

Reply:
#111 also isn't really about unit testing, even if that's where it currently gives us the most benefit.

Reply:
#112 is also not about unit testing, it's a performance improvement (albeit for a command that's not recommended outside of testing use cases).

Reply:
#116 is mostly a fix for problems in #111. So I'd add a separate item that says something like
```
* Improve `wait_keys` responsiveness for `MemoryBackend` (#111, #116)
```

Reply:
I did try to put this into as few lines as possible, focussing on the take-home message for external users (if there are even any - let's call them less informed users :-)). #107 only affects unit tests and #112 only makes sense in unit tests, so I took some liberties here.

How about this split?
```
* Improve `wait_keys` responsiveness for `MemoryBackend` (#111, #116)
* Speed up telstate.clear and support older HMSET command in unit tests (#107, #112)
```

Reply:
I think I misunderstood what #107 was about - you're right that it only affects unit tests.

I don't think #107 and #112 belong on the same line though. #107 is about telstate's unit tests, #112 is about using telstate in your own code (and generally *not* in unit tests, since in unit tests you'd use a memory backend). Replace the 'and' with a line break (and distribute the PR numbers accordingly) and I'm happy.

Reply:
I've done this. Have we ever called `telstate.clear()` on a Redis server outside of unit tests?

Reply:
[Yes](https://github.com/ska-sa/katsdpcal/blob/21cdd493911c4145b092b3531c8cb7c770623b2a/scripts/sim_ts.py#L36).